### PR TITLE
Add claims mentors index for support and normal users

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Context

This PR adds the index page for mentors for normal and support users. In the claims doamin

## Changes proposed in this pull request

- Mentors index page for normal user
- Mentors index page for support users
- Fix seed file

## Guidance to review

Visit `claims.localhost:3000/schools/:school_id/mentors`
Visit `claims.localhost:3000/support/schools/:school_id/mentors` as support user

## Screenshots

https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/0ff6b44f-2057-4e9e-9472-6dc0a3a7783e

### CODE REVIEW

File: app/views/claims/support/claims/index.html.erb [hunk 1]
```diff
@@ -6,33 +6,34 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
 
-    <% if @claims.any? %>
-      <%= govuk_table do |table| %>
-        <% table.with_head do |head| %>
-          <% head.with_row do |row| %>
-            <% row.with_cell(header: true, text: Claim.human_attribute_name(:id)) %>
-            <% row.with_cell(header: true, text: t(".status")) %>
+      <% if @claims.any? %>
```
Comment (CatalinVoineag) [inline app/views/claims/support/claims/index.html.erb:9]: The indentation was a bit off here

**Comment by CatalinVoineag** [comment]
[As a Support User, I want to see a list of all Mentors belonging to a School](https://trello.com/c/DrThT8ao/193-as-a-support-user-i-want-to-see-a-list-of-all-mentors-belonging-to-a-school)

**Comment by CatalinVoineag** [comment]
[As a User, I want to see a list of all Mentors belonging to a School](https://trello.com/c/OER4DRum/213-as-a-user-i-want-to-see-a-list-of-all-mentors-belonging-to-a-school)

File: app/views/claims/schools/mentors/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<% render "claims/schools/primary_navigation", school: @school, current: :mentors %>
```
Comment (JamieCleare2525) [inline app/views/claims/schools/mentors/index.html.erb:1]: This page doesn't have a title.

File: app/views/claims/support/schools/mentors/index.html.erb [hunk 3]
```diff
@@ -8,6 +8,35 @@
   <div class="govuk-grid-row">
```
Comment (JamieCleare2525) [inline app/views/claims/support/schools/mentors/index.html.erb:8]: Could you add a title to this page too?

File: db/seeds.rb [hunk 4]
```diff
@@ -45,31 +45,31 @@
 # Associate Placements Users with Organisations
 # Single School Anne
 placements_anne = Placements::User.find_by!(email: "anne_wilson@example.org")
-placements_anne.memberships.find_or_create_by!(organisation: Placements::School.first)
+placements_anne.user_memberships.find_or_create_by!(organisation: Placements::School.first)
```
Comment (JamieCleare2525) [inline db/seeds.rb:48]: Great catch!

File: app/views/claims/support/schools/mentors/index.html.erb (hunk 3 above)
Comment (CatalinVoineag) [reply app/views/claims/support/schools/mentors/index.html.erb:9]: Good catch, I've added it

File: app/views/claims/support/schools/mentors/index.html.erb [hunk 5]
```diff
@@ -8,6 +9,35 @@
...
     <div class="govuk-grid-column-two-thirds">
       <h2 class="govuk-heading-m"><%= t(".heading") %></h2>
+
+      <% if @mentors.any? %>
+        <%= govuk_table do |table| %>
+          <% table.with_head do |head| %>
+            <% head.with_row do |row| %>
+              <% row.with_cell(header: true, text: t(".name")) %>
+              <% row.with_cell(header: true, text: t(".trn")) %>
```
Comment (Nitemaeric) [inline app/views/claims/support/schools/mentors/index.html.erb:18]: Same as above.

**Review by Nitemaeric — COMMENTED** [review]
Some minor comments :)

File: app/views/claims/schools/mentors/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".heading") %></h2>
+
+      <% if @mentors.any? %>
+        <%= govuk_table do |table| %>
+          <% table.with_head do |head| %>
+            <% head.with_row do |row| %>
+              <% row.with_cell(header: true, text: t(".name")) %>
+              <% row.with_cell(header: true, text: t(".trn")) %>
```
[the remark this answers is not in the document]
Comment (CatalinVoineag) [reply app/views/claims/schools/mentors/index.html.erb:14]: The trn translation is `Teacher reference number (TRN)`. Can to this for the name though

File: db/seeds.rb (hunk 4 above)
Comment (CatalinVoineag) [reply db/seeds.rb:48]: Haha, no worries 👍🏻

File: db/seeds.rb
[the remark this answers is not in the document]
Comment (CatalinVoineag) [reply db/seeds.rb:1]: https://github.com/DFE-Digital/itt-mentor-services/pull/275/commits/aa86c8b438f85f8e08e6c7f9cc5695d6e1e28c37

File: app/views/claims/support/schools/mentors/index.html.erb (hunk 5 above)
Comment (CatalinVoineag) [reply app/views/claims/support/schools/mentors/index.html.erb]: https://github.com/DFE-Digital/itt-mentor-services/pull/275/commits/dfc33c141acfb6d2a2e66a40163b278f09d0078c

File: app/views/claims/support/claims/index.html.erb [hunk 7]
```diff
@@ -6,33 +6,34 @@
...
+        <%= govuk_table do |table| %>
+          <% table.with_head do |head| %>
+            <% head.with_row do |row| %>
+              <% row.with_cell(header: true, text: Claim.human_attribute_name(:id)) %>
+              <% row.with_cell(header: true, text: t(".status")) %>
```
[the remark this answers is not in the document]
Comment (CatalinVoineag) [reply app/views/claims/support/claims/index.html.erb]: https://github.com/DFE-Digital/itt-mentor-services/pull/275/commits/dfc33c141acfb6d2a2e66a40163b278f09d0078c

File: app/views/claims/schools/mentors/index.html.erb (hunk 6 above)
[the remark this answers is not in the document]
Comment (CatalinVoineag) [reply app/views/claims/schools/mentors/index.html.erb]: https://github.com/DFE-Digital/itt-mentor-services/pull/275/commits/dfc33c141acfb6d2a2e66a40163b278f09d0078c

[the remark this answers is not in the document]
Comment (Nitemaeric) [reply app/views/claims/schools/mentors/index.html.erb]: I think that's fine actually. We should add an explicit translation for name at `activerecord.yml` too though.

**Review by Nitemaeric — APPROVED** [review]

**Review by JamieCleare2525 — APPROVED** [review]
LGTM!

**Pull request merged**